### PR TITLE
Allow for a test to define the backend which it'll run under, if avai…

### PR DESCRIPTION
**Pull request by geoffjentry**
…lable

### CODE REVIEW

File: src/main/resources/standardTestCases/refresh_token.test [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: refresh_token
 testFormat: workflowsuccess
+backend: JES
```
Comment (ruchim) [inline src/main/resources/standardTestCases/refresh_token.test]: this might have to be "jes-refresh" as that's what the backend is called in the .conf

**Comment by ruchim** [comment]
You probably already know this--I had some spray errors when trying to use this branch.  The errors were fairly non specific: 

```
[info] Exception encountered when attempting to run a suite with class name: org.scalatest.DeferredAbortedSuite *** ABORTED ***
[info]   spray.httpx.UnsuccessfulResponseException: Status: 404 Not Found
[info] Body: The requested resource could not be found.
[info]   at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:39)
[info]   at spray.httpx.ResponseTransformation$$anonfun$unmarshal$1.apply(ResponseTransformation.scala:31)
[info]   at scala.util.Success$$anonfun$map$1.apply(Try.scala:237)
[info]   at scala.util.Try$.apply(Try.scala:192)
[info]   at scala.util.Success.map(Try.scala:237)
[info]   at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:237)
[info]   at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:237)
[info]   at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32)
[info]   at scala.concurrent.impl.ExecutionContextImpl$AdaptedForkJoinTask.exec(ExecutionContextImpl.scala:121)
[info]   at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
[info]   ...``
```

EDIT: ran against the wrong Cromwell!

**Comment by ruchim** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/99/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/refresh_token.test (hunk 1 above)
Comment (geoffjentry) [inline src/main/resources/standardTestCases/refresh_token.test]: You're right, however that conf file doesn't seem to be used at the moment. let's chat in person, i'm probably confused

**Comment by geoffjentry** [comment]
@mcovarr Apologies for already squashing, I thought you'd already thumbsd this

**Comment by geoffjentry** [comment]
@mcovarr still waiting on some thumbage

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/99/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**